.custom_wordlist.txt

```diff
@@ -18,6 +19,9 @@ AMIs
 Anbox
 Ansible
 Anthos
+apparmor
+apport
+appstream
```

Comment:
All three of them are in a code-block, so they are not needed here.

Reply:
They popped up in the spell check, but must have been before I added them to code blocks. Thanks, removed all words that appear in code blocks.